wger/settings_global.py

```diff
@@ -159,6 +167,9 @@
 SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = os.environ.get("SOCIAL_AUTH_GOOGLE_OAUTH2_KEY")
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = os.environ.get("SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET")
 
+# Guardian permission denied setting
+GUARDIAN_RAISE_403 = True
+
```

Comment:
Consider setting `ANONYMOUS_USER_NAME` to `None` to avoid refactoring the failing tests.

> The Guardian anonymous user is different to the Django Anonymous user. The Django Anonymous user does not have an entry in the database, however the Guardian anonymous user does.